src/unit_tests/logcollector/test_logcollector.c

```diff
@@ -863,17 +852,14 @@ void test_w_load_files_status_update_add_fail(void ** state) {
     will_return(__wrap_OS_SHA1_File_Nbytes, "32bb98743e298dee0a654a654765c765d765ae80");
     will_return(__wrap_OS_SHA1_File_Nbytes, 1);
 
-    will_return(__wrap_OSHash_Update_ex, 0);
-
-    expect_value(__wrap_OSHash_Add_ex, self, files_status);
-    expect_string(__wrap_OSHash_Add_ex, key, file);
-    will_return(__wrap_OSHash_Add_ex, 0);
+    expect_function_call(__wrap_pthread_rwlock_wrlock);
+    expect_function_call(__wrap_pthread_rwlock_unlock);
 
     expect_string(__wrap__merror, formatted_msg, "(1298): Failure to add 'test' to 'file_status' hash table");
 
     w_load_files_status(global_json);
-
 }
+*/
```

Comment:
Same as with the previous commented test, don't make any extra changes to it, just comment it out.